problems/arrays/problem_es.md

```diff
@@ -12,7 +12,7 @@ var pets = ['cat', 'dog', 'rat'];
 
 Crea un archivo llamado `arrays.js`
 
-En ese archivo define una variable llamada `pizzaToppings` que referencie a un array el cual contenga los siguientes elementos (en el siguiente orden): `tomato sauce, cheese, pepperoni`.
+En ese archivo define una variable llamada `condimentos` que referencie a un array el cual contenga los siguientes elementos (en el siguiente orden): `tomato sauce, cheese, pepperoni`.
```

Comment:
This `pizzaToppings` is intentional. see #104 
Maybe you can fix other `condimentos` instead.